File: app/src/main/java/com/example/volumeintimemanager/IAddRule.java
package com.example.volumeintimemanager;

import android.view.View;

public interface IAddRule {
    String checkDigit(int number);
    void dayPicker(View view);
    String replaceDaysIDsWithDaysNames(String daysIds);
    void addRuleButton(View view);
    void loadSpinnerItems();
}

File: app/src/main/java/com/example/volumeintimemanager/IAddRule.kt
package com.example.volumeintimemanager

import android.view.View

interface IAddRule {
    fun checkDigit(number: Int): String?
    fun dayPicker(view: View?)
    fun replaceDaysIDsWithDaysNames(daysIds: String?): String?
    fun addRuleButton(view: View?)
    fun loadSpinnerItems()
}